Framework/DataHandling/src/LoadAscii.cpp

```diff
@@ -177,7 +177,9 @@ API::Workspace_sptr LoadAscii::readData(std::ifstream &file) const {
   // could be blank lines and comment lines
   int numBins(0), lineNo(0);
   std::vector<DataObjects::Histogram1D> spectra(
-      numSpectra, HistogramData::Histogram::XMode::Points);
+      numSpectra,
+      DataObjects::Histogram1D(HistogramData::Histogram::XMode::Points,
+                               HistogramData::Histogram::YMode::Counts));
```

Comment:
I much prefer this explicit `YMode` specification. I think this is a really good step to forcing people to think about the nature of their data at the point of import.